content/sensu-go/6.8/api/other/ready.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "/ready"
+description: "Read this API documentation for information about Sensu /ready API endpoints, including examples for learning whether the Sensu instance is ready to serve API requests."
+other_api_title: "/ready"
+type: "other_api"
+version: "6.8"
+product: "Sensu Go"
+menu:
+  sensu-go-6.8:
+    parent: other
+---
+
+## Get API readiness data for your Sensu instance
+
+The `/ready` API endpoint provides HTTP GET access to information about whether your Sensu instance is ready to serve API requests.
+
+### Example
+
+The following example demonstrates a GET request to the `/ready` API endpoint:
+
+{{< code shell >}}
+curl -X GET \
+http://127.0.0.1:8080/ready
+{{< /code >}}
+
+The request results in a successful `HTTP/1.1 200 OK` response and a text response body:
+
+{{< code text >}}
+ready
+{{< /code >}}
+
+If the backend configuration includes an [`api-serve-wait-time`][2] duration, the request will result in an `HTTP/1.1 503 Service Unavailable` response.
+Until the `api-serve-wait-time` duration expires, the text response body will state that the API is unavailable:
+
+{{< code text >}}
+API unavailable during startup.
+See api-serve-wait-time settings.
+{{< /code >}}
+
+{{% notice note %}}
+**NOTE**: `503 Service Unavailable` responses include a `Retry-After` header that lists the specified `api-serve-wait-time` duration.
+{{% /notice %}}
+
+### API Specification
+
+/ready (GET)    | 
+-----------------|------
+description      | Returns information about whether the Sensu instance is ready to serve API requests.
+example url      | http://hostname:8080/ready
+response type    | text
+response codes   | <ul><li>**Success**: 200 (OK)</li><li>**Error**: 503 (Service Unavailable)</li></ul>
+output           | 200 (OK):<br><br>{{< code text >}}
+ready
+{{< /code >}}<br>503 (Service Unavailable):<br><br>{{< code text >}}
+API unavailable during startup.
+See api-serve-wait-time settings.
+{{< /code >}}
+
+## Get agent connection readiness data for your Sensu instance
+
+The `/ready` agent transport API endpoint provides HTTP GET access via the backend WebSocket to information about whether your Sensu agent transport is ready to accept agent connections.
```

Comment:
I'd suggest rewording this just a bit or just dropping the "via the backend WebSocket" part.
It isn't a websocket request - it just happens to be served on the same http service that we use to accept agent websocket connections. It looks like we call this service the "agent listener" in most other places.

Maybe something like
> The `/ready` agent listener API endpoint provides HTTP GET access to information about whether your Sensu agent transport is ready to accept agent WebSocket connections.

Reply:
Fixed it, thank you!